Upstream-Status: Pending

We should make sure that sysroot is used by gcc instead of assuming
hardcoded locations for include paths

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 src/drivers/drivers.mak | 2 +-
 src/drivers/drivers.mk  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/drivers/drivers.mak b/src/drivers/drivers.mak
index 7e175f4..950701f 100644
--- a/src/drivers/drivers.mak
+++ b/src/drivers/drivers.mak
@@ -35,7 +35,7 @@ NEED_RFKILL=y
 ifdef CONFIG_LIBNL32
   DRV_LIBS += -lnl-3
   DRV_LIBS += -lnl-genl-3
-  DRV_CFLAGS += -DCONFIG_LIBNL20 -I/usr/include/libnl3
+  DRV_CFLAGS += -DCONFIG_LIBNL20 -I=/usr/include/libnl3
 else
   ifdef CONFIG_LIBNL_TINY
     DRV_LIBS += -lnl-tiny
diff --git a/src/drivers/drivers.mk b/src/drivers/drivers.mk
index db8561a..c93e88d 100644
--- a/src/drivers/drivers.mk
+++ b/src/drivers/drivers.mk
@@ -30,7 +30,7 @@ NEED_RFKILL=y
 ifdef CONFIG_LIBNL32
   DRV_LIBS += -lnl-3
   DRV_LIBS += -lnl-genl-3
-  DRV_CFLAGS += -DCONFIG_LIBNL20 -I/usr/include/libnl3
+  DRV_CFLAGS += -DCONFIG_LIBNL20 -I=/usr/include/libnl3
 else
   ifdef CONFIG_LIBNL_TINY
     DRV_LIBS += -lnl-tiny
-- 
2.1.2

